src/merkledag.proto.js

```diff
@@ -1,3 +1,6 @@
+'use strict'
+
+module.exports = new Buffer(`
```

Comment:
It removes a lot of complexity for users, not using aegir. These proto files are the reason we need to use brfs as a preprocessor everywhere. If we drop it the code works out of the box in webpack and browserify.